src/test/java/com/exasol/adapter/dialects/exasol/ExasolFromExaQueryRewriterTest.java

```diff
@@ -54,7 +54,7 @@ void testRewriteLocal() throws AdapterException, SQLException, ExaConnectionAcce
         setIsLocalProperty();
```

Comment:
ExaConnectionAccessException is never thrown in this method

Reply:
Eclipse claims it is.

Reply:
Fixed.